chromiumcontent/args/ffmpeg.gn

```diff
@@ -3,11 +3,15 @@ is_component_build = false
 is_mas_build = getenv("MAS_BUILD") != ""
 is_debug = false
 enable_nacl = false
-enable_widevine = true
 proprietary_codecs = false
 is_component_ffmpeg = true
 ffmpeg_branding = "Chromium"
 
+# As of Chromium 67.0.3396.99 widevine is only supported on Mac and Windows.
+if (target_os != "linux") {
+  enable_widevine = true
+}
```

Comment:
This seems like a bummer; is upstream no longer supporting this?

Reply:
We don't have the chrome component updater setup in electron, also for proprietary reasons we ask users to get their own widevine libraries, electron only integrates mechanism required to load cdm. So this build flag will not affect us directly.

Reply:
@nornagon It is not supported on Linux in Ch67. I have no idea if there are plans to support it in the future.